engine/execution/utils/hasher.go

```diff
@@ -22,6 +22,12 @@ func NewSPOCKHasher() hash.Hasher {
 	return h
 }
 
+// NewDKGMessageHasher returns a hasher for signing and verifying DKG broadcast
```

Comment:
Probably makes sense to move this to a common package. This is under `engine/execution`, but the execution node doesn't really have anything to do with the DKG. Maybe we can move this file to `utils/hasher`